meta-luneos/recipes-core/packagegroups/packagegroup-webos-extended.bb

```diff
@@ -129,3 +130,7 @@ RDEPENDS_${PN}_append_qemuall = " \
 # Unused meta-webos components:
 # - glibcurl
 # - libtinyxml
+
+PDM = " \
+    com.webos.service.pdm \
+"
```

Comment:
If it was to follow what OSE did, then you forget to include also this part:

# PDM service functionality cannot be verified on webOS rpi64 which cannot boot yet
PDM_raspberrypi3-64 = ""

Reply:
Well rpi64 doesn't boot, unlikely due to PDM I'd say. I suggest we build for all ;)

Reply:
Changed